docs/Modules/bpmn-modules/modeling.md

```diff
@@ -9,4 +9,160 @@ parent: "BPMN Modules"
 grand_parent: "Modules"
 ---
 
-# modeling
\ No newline at end of file
+# modeling
+
+This module activates the BPMN 2.0 modeling features [over the common modeling module](/modules/common-modules/modeling). It means that this module contains every API exported by the common modeling module.
+
+Additional APIs over the common modeling module are:
+
+* [updateLabel](#updatelabel)
+* [connect](#connect)
+* [updateProperties](#updateproperties)
+* [resizeLane](#resizelane)
+* [addLane](#addlane)
+* [splitLane](#splitlane)
+* [setColor](#setcolor)
+
+## updateLabel
+
+`updateLabel` API updates labels and changes the names of elements. This API fires
+[elements.changed](/events/common-events/elements-changed) event and executes [element.updateLabel](/commands/bpmn-commands/element-updateLabel) command.
+
+### Usage
+
+```javascript
+const elementRegistry = bpmnJS.get('elementRegistry');
+const modeling = bpmnJS.get('modeling');
+
+const startEvent_1 = elementRegistry.get('StartEvent_1');
+
+modeling.updateLabel(startEvent_1, 'new label');
+```
+
+## connect
+
+`connect` API performs the same job as [the common createConnection API](/modules/common-modules/modeling#createconnection). The only difference is that `connect` API first checks if the source element and the target element are connectable by BPMN standards before delegating to `createConnection` API.
+
+As this API delegates to the common `createConnection` API, this executes every command/fires every event that the common API executes/fires.
+
+### Usage
+
+```javascript
+const elementRegistry = bpmnJS.get('elementRegistry');
+const modeling = bpmnJS.get('modeling');
+
+const sourceShape = elementRegistry.get('Task_1');
+const targetShape = elementRegistry.get('Task_2');
+
+modeling.connect(sourceShape, targetShape);
+
+```
+
+To create connection with custom start/end points:
+```javascript
+modeling.connect(sourceShape, targetShape, null, {
```

Comment:
As a user, I now wonder what the "null" parameter is about. How to know what parameters to fill inside the API method?

Reply:
Agreed. We shouldn't use `null` without any explanation.

Could be changed to

```javascript
const attrs = null;

modeling.connect(sourceShape, targetShape, attrs, { ... });
```

Reply:
Updated the example, also now the input/output params of each API is included so that'd not be a problem anymore.